### PR TITLE
Add script for searching trough abstracts

### PR DESCRIPTION
**Pull request by madbkr**
This script is meant to help with the oekg evaluation. 

It takes the data file containing the graph with the abstracts and a textfile containing all oeo labels and checks the abstracts for them.

### CODE REVIEW

File: oekg evaluation/How to use.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This script is calles like this:
```
Comment (stap-m) [inline oekg evaluation/How to use.txt]:
```suggestion
This script is called like this: 
```

File: oekg evaluation/How to use.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+This script is calles like this: 
+python3 wordFinder.py [parth to your rdf data file] [path to the file holding labels].
```
Comment (stap-m) [inline oekg evaluation/How to use.txt]:
```suggestion
python3 wordFinder.py [path to your rdf data file] [path to the file holding labels].
```

File: oekg evaluation/How to use.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+The output will be printed into a file called results.txt.
+You may need to instal rdflib first.
+
+The file with all current labels taken from the glossary and current alternative labels extracted from the omn files is provided in this folder.
```
Comment (stap-m) [inline oekg evaluation/How to use.txt]:
```suggestion
The file with all current OEO labels taken from the glossary and current alternative labels extracted from the omn files is provided in this folder.
```

**Comment by stap-m** [comment]
Great thanks for the scripts! 
Do you use a separate script which extracts the alternative labels or did you do that manually?

**Comment by madbkr** [comment]
> Great thanks for the scripts! Do you use a separate script which extracts the alternative labels or did you do that manually?

I made a script for that too. If you want I'll add it here (just need to clean it up a bit).

**Comment by stap-m** [comment]
As far as I understand, the output counts how often a label appears in all studies? Could you store the study URI together with each label found?

**Comment by madbkr** [comment]
> As far as I understand, the output counts how often a label appears in all studies? Could you store the study URI together with each label found?

No, it prints a study URI and then how many times the words were found in this abstract.

like this:
```
http://openenergy-platform.org/ontology/oekg/scenario/3fc42966-9510-7148-c6ed-a21f79a86636
energy: 1
policy: 1
scenario: 2
```

**Review by stap-m — APPROVED** [review]

**Comment by madbkr** [comment]
@stap-m 

I added the scripts I used for label extraction. Maybe you want to also review those before I merge.

**Review by stap-m — APPROVED** [review]

**Pull request merged**